split-webapp/split/src/pages/Split/PeopleList.module.css

```diff
@@ -0,0 +1,193 @@
+.peopleList::before {
+  display: block;
+  margin-top: -1px;
+  content: "";
+  height: 0px;
+}
+
+/**
+ * Section:
+ * Row and drag-and-drop container for each row
+ */
+
+.personRow {
+  display: flex;
+  height: 44px;
+  margin-bottom: -1px;
+  align-items: center;
+  border-top: 1px solid #d6e3fe;
+  border-bottom: 1px solid #d6e3fe;
+
+  /* We don't want them to be see-through when drag-and-dropping */
+  background-color: white;
+}
+
+.peopleListDragHelper {
+  list-style: none;
+  z-index: 3;
+}
+
+.peopleListDragHelper > .personRow {
+  width: calc(100% + 1em - 2px); /* 2px is to account the border */
+  margin: 0 -0.5em;
+  border: 1px solid #d6e3fe;
+  box-shadow: 0 0 6px rgba(0, 0, 0, 0.16);
+  cursor: grabbing;
+}
+
+.peopleListDragHelper .errorBubble {
+  opacity: 0;
+}
+
+/**
+ * Section:
+ * Name input text-field
+ */
+
+.personNameWrapper {
+  display: flex;
+  flex-grow: 1;
+}
+
+.personNameInput {
+  /* Fix minor clipping of descenders like 'g' */
+  padding: 0.1em !important;
+
+  /* Account for the border's thickness for when it appears. */
+  /* Also set up initial values for transition. */
+  border-bottom: 1px solid transparent !important;
+
+  transition: border 0.3s;
+  color: #2d4583 !important;
+  font-size: 14px !important;
+}
+
+.personNameInput::placeholder {
+  opacity: 1 !important;
+  color: #afc6fd !important;
+  font-weight: 300 !important;
+  font-style: italic !important;
+}
+
+.personNameInput:focus::placeholder {
+  opacity: 0.4 !important;
+}
+
+.personRow:hover .personNameInput {
+  border-bottom-color: #afc6fd !important;
+}
+
+.personNameInput:focus,
+.personRow:hover .personNameInput:focus {
+  border-bottom-color: #d6e3fe !important;
+}
+
+.personNameWrapper .errorBubble {
+  top: calc(100% - 10px);
+}
+
+/**
+ * Section:
+ * Toggle buttons
+ */
+
+.personPayeeToggleGroup {
+  margin: 0 20px 0 30px;
+  display: flex;
+}
+
+.personPayeeToggleGroup > button {
+  height: 22px;
+  transition: background 0.3s, border 0.3s, color 0.3s;
+  line-height: 1;
+  text-transform: none !important;
+}
+
+.personPayeeToggleGroup .personIsPayerButton {
+  border-color: #d6e3fe;
+  color: #d6e3fe;
+}
+
+.personPayeeToggleGroup .personIsPayeeButton {
+  border-color: #ffd8f1;
+  color: #ffd8f1;
+}
+
+.personPayeeToggleGroup .personIsPayerButton:hover,
+.personPayeeToggleGroup .personIsPayerButtonSelected {
+  border-color: #afc6fd;
+  background-color: #afc6fd !important;
+  color: white !important;
+}
+
+.personPayeeToggleGroup .personIsPayeeButton:hover,
+.personPayeeToggleGroup .personIsPayeeButtonSelected {
+  border-color: #e95eb1;
+  background-color: #e95eb1 !important;
+  color: white !important;
+}
+
+.personPayeeWrapper .errorBubble {
+  top: calc(100% + 10px);
+}
+
+/**
+ * Section:
+ * Person row action icons / buttons
+ */
+
+.dragHandle,
+.personRemoveButton {
+  height: 100%;
+  min-width: 44px !important;
+  border-radius: 0 !important;
+  transition: color 0.3s !important;
+  color: #d6e3fe !important;
```

Comment:
There seems to be an awful lot of `!important` in the css files, wasn't there a way to change the order of css injection/priority with material ui? Or is there some other way to avoid using it?

Reply:
Yeah, I don't like the `!important` either. I'll see if there's a way to change the injection order.

Reply:
Yikes, using `<StyleProvider injectFirst>` to make material-ui's style lesser priority than our css appears to break a lot of the app's styles.

![image](https://user-images.githubusercontent.com/2609018/80162824-7e0fcd00-8628-11ea-989c-c604108f1a3a.png)

I think we should leave things as is for now. I'll create a new issue/story for this instead.

Reply:
Sounds good!